torch/_inductor/template_heuristics.py

```diff
@@ -707,6 +707,18 @@ class CUDAConfigHeuristic(BaseConfigHeuristic):
     def __init__(self) -> None:
         super().__init__()
 
+        self.b200_default_flex_config = {
+            (torch.float32, 64): FlexConfig(128, 32, 3, 4),
```

Comment:
any method to this madness? or seemed to work?

Reply:
Seems fine for now, at least from my initial investigation flex perf on b200 is pretty bad so getting it to work is fine for now and can come back to tuning later